### PR TITLE
Update the existing vignette to incorporate changes from PR #88.

### PR DESCRIPTION
**Pull request by monicagerber**
The changes to this vignette reflect the changes in PR #88. Can you please let me know if these updates are clear and easy to understand? Is there anything else you'd like to know about using a VISC report template?

### CODE REVIEW

**Comment by monicagerber** [comment]
@wfulp @mayerbry @celiaeddy please let me know if you have any feedback by end of day 12/14 or earlier. Thanks!

**Review by wfulp — APPROVED** [review]
This looks great and runs fine. 

One side note is I think you need to make the `fs` package a dependency, since it's used in the `challenge_directory` function.

**Comment by monicagerber** [comment]
Thanks for your review @wfulp! I believe I added fs as a dependency on another PR to the develop branch. I'll make sure it happens. 😄

File: vignettes/using_pdf_and_word_template.Rmd [hunk 1]
```diff
@@ -64,13 +102,13 @@ There are some new functions and coding conventions that help with knitting the
 * Note that Word output cannot support many kableExtra features, such as `kableExtra::kable_styling()`.
    + Set `warning=kable_warnings` chunk option, otherwise Word output will have unwanted warnings.
 
-## Figures
+### Figures
 
 * Only output one figure per code chunk.
 * Figure name will be chunk name plus "fig:".
     + i.e. if chunk name is "-stats" then the table label will be "fig:descriptive-stats".
```
Comment (mayerbry) [inline vignettes/using_pdf_and_word_template.Rmd:109]: `i.e. if chunk name is “-stats” then the table label will be “fig:descriptive-stats”.`
"descriptive-stats"

**Review by mayerbry — APPROVED** [review]
Looks great to me! I rely on the README in VISCtemplates a lot, so I hope a brief updated version of this stays there when that gets updated (with a reference to this vignette).

I found a small typo that I tagged.

**Comment by monicagerber** [comment]
Thanks @mayerbry ! I did change the package README a bit by moving some of the details to a vignette. See PR #90 , comments welcome. The README still reminds users how to use the two main functions of the package.

It's also on my to-do list to look into creating a pkgdown site for VISCtemplates. That would make the vignettes easier to access.

**Pull request merged**